snorkel/labeling/model/label_model.py

```diff
@@ -801,6 +803,14 @@ def fit(
             # Update learning rate
             self._update_lr_scheduler(epoch)
 
+        # Clamp learned parameters
+        # Note: If mu_clamp_min is set too high, e.g. in sparse settings where LFs
+        # mostly abstain, this will cause null results!
+        self.mu.data = self.mu.clamp(  # type: ignore
```

Comment:
Why do this here (and outside of the computation graph) instead of directly at inference time?

Reply:
Regarding doing it here- I view this as part of our current procedure for learning / setting `mu`, therefore would go here.  Also safer if mu used in multiple sub-functions.

As for outside the computation graph- `torch.clamp` returns a tensor not a parameter, that's why I did this